apps/coscrad-frontend/src/components/resources/digital-text/full-page-presenter.tsx

```diff
@@ -0,0 +1,47 @@
+import { IDigitalTextPage } from '@coscrad/api-interfaces';
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import { Typography, styled } from '@mui/material';
+import { MultilingualTextPresenter } from '../../../utils/generic-components/presenters/multilingual-text-presenter';
+import { PageContentForm, TextAndLanguage } from './page-content-form';
+
+const StyledMuiPage = styled('div')({
+    width: '140px',
+    height: '200px',
+    margin: '2px',
+    position: 'relative',
+    float: 'left',
+    border: '2px solid #05803b',
+});
+
+interface FullPagePresenterProps {
+    page: IDigitalTextPage;
+    isSelected: boolean;
+    onSubmitNewContent: (state: TextAndLanguage) => void;
+    isAdmin?: boolean;
+}
+
+export const FullPagePresenter = ({
+    page,
+    isSelected,
+    onSubmitNewContent,
+    isAdmin: _isAdmin = false,
+}: FullPagePresenterProps): JSX.Element => {
+    const { identifier, content } = page;
+
+    const hasContent = !isNullOrUndefined(content);
+
+    const shouldShowAddContentForm = !hasContent; // && isAdmin
+
+    return (
+        <StyledMuiPage data-testid={`digital-text.page:${identifier}`}>
```

Comment:
This needs some of Geoff's work. Instead of a page "icon", we want to fill up as much of the screen real estate as possible to allow space to work.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186539848) is the relevant story.